libs/product/package.json

```diff
@@ -12,9 +12,8 @@
     "url": "https://github.com/graycoreio/daffodil/issues"
   },
   "scripts": {
-    "build": "ng build product",
-    "lint": "ng lint product",
-    "test": "ng test product --watch=false --browsers=ChromeHeadless"
+    "build": "ng build --prod",
```

Comment:
`ng build product --prod`

Reply:
Yeah, I thought that was odd, but I thought I'd take your word for it

Reply:
done